istio-control/istio-discovery/templates/autoscale.yaml

```diff
@@ -1,16 +1,17 @@
-{{- if .Values.autoscaleMin }}
+{{- if and .Values.autoscaleEnabled .Values.autoscaleMin .Values.autoscaleMax }}
 apiVersion: autoscaling/v2beta1
 kind: HorizontalPodAutoscaler
 metadata:
   name: istio-pilot
   namespace: {{ .Release.Namespace }}
   labels:
+    app: pilot
     release: {{ .Release.Name }}
```

Comment:
can you file an AC issue for 1.2 to purge all unnecessary labels from the installer codebase?  I know for now they help with diffing.